internal/rbd/controllerserver.go

```diff
@@ -592,7 +592,6 @@ func checkContentSource(ctx context.Context, req *csi.CreateVolumeRequest, cr *u
 // DeleteVolume deletes the volume in backend and removes the volume metadata
 // from store
 // TODO: make this function less complex
```

Comment:
Dropping the next line causes golint to fail with:
```
Top level comment should end in a period (godot)
```
That means, the previous line should end in a `.`:
```
// TODO: make this function less complex.
```

Need to adjust that here, and on other places.